README.md

```diff
@@ -290,6 +290,35 @@ jobs:
 ```
 </details>
 
+### TestIO
+
+<details>
+
+<summary>This GitHub Action can be used to trigger a test on the external crowd-testing platform TestIO from a pull request.</summary>
+
+```yml
+name: TestIO - Trigger test from PR
+on:
+  issue_comment:
+    types: [created, edited]
```

Comment:
I guess there is also some logic that you don't spam the TestIO people? 🤔

Reply:
We pay good money 💸 So I don't get your point 😉